templates/home/dashboard.html

```diff
@@ -322,7 +325,7 @@ <h4 class="text-center mt-3">Volunteers Present</h4>
     if(document.getElementById("extra-info-1").value.length>0)
         $('#class-date').datepicker("setDate", "{{ selected_date|date:'Y-m-d' }}");
     else
-        $('#class-date').datepicker("setDate", new Date());
+        $('#class-date').datepicker("setDate", current_date);
```

Comment:
I think you also need to convert `current_date` to a `Date` object. Not sure though!

Reply:
Just saying current_date worked. Should I convert it into a Date() object?

Reply:
Oh! Alright then.
In that case, an even better way to solve this would be to put `{% now 'Y-m-d' %}` directly in place of `new Date()`. Why create and use an extra variable, right?

Reply:
Done.